src/transport/raw/tests/NetworkTestHelpers.cpp

```diff
@@ -29,6 +29,10 @@ namespace Test {
 
 CHIP_ERROR IOContext::Init()
 {
+#if defined(CHIP_DEVICE_LAYER_TARGET_OPEN_IOT_SDK)
+    ReturnErrorOnFailure(chip::DeviceLayer::PlatformMgr().InitChipStack());
```

Comment:
why was this not needed before and needed now? It seems like stack handling should be closer to some form of "main" rather than a IOContext sideffect.

Reply:
Fixed